app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -25,15 +25,14 @@ PROVIDES="
 
 REQUIRES="
 	haiku$secondaryArchSuffix
-	guile1$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libQt5Concurrent$secondaryArchSuffix
 	lib:libQt5Core$secondaryArchSuffix
 	lib:libQt5Gui$secondaryArchSuffix
 	lib:libQt5Svg$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
-	lib:libguile$secondaryArchSuffix
+	lib:libguile$secondaryArchSuffix <= 19
```

Comment:
Guile 1.8.x recipe says:
```libguileVersion="17.4.0"```, which becomes 17 as compat, so why are you constraining it to 19?

Reply:
(see below)

Reply:
I've changed to == 17.4.0